docs/installation/migrate-to-xcp-ng.md

```diff
@@ -135,11 +135,41 @@ You need to rescan the SR where you new VHD file is, so it can be detected. It w
 
 There's two options, both requiring to export your Hyper-V VM disk in VHD format.
 
-:::warning
-When exporting in VHD, always use a **dynamic disk** VHD format and not **static**, which doesn't work in XCP-ng.
-:::
+### Export the VM disk
+
+**If the server can be taken offline:** Shut down the VM and create a VHD file from the existing VHDX.  
+  This process leaves the original disk file unchanged, allowing you to restart the VM in Hyper-V if needed. Ensure sufficient disk space is available for both the original VM and the new VHD file.
+
+**If the server must remain online:** Export the VM and then convert the VHDX to a VHD file.  
+  Note that the original VM will continue running and may be updated during the migration process. Ensure enough disk space is available for the original VM, the exported VM, and the new VHD file.
+
+1. Prepare the VHD for export.  
+  Before exporting, remove all the Hyper-V tools from the VM to ensure compatibility.
+
+2. (Optional). Shut down the VM in Hyper-V.  
+  To shut down the VM, run this command in PowerShell:
+  ```powershell
+  STOP-VM -Name <VM name>
+  ```
+
+3. Identify the VM disk to be exported.  
+  To identify the VM disk, run this command:
+  ```powershell
+  Get-VMHardDiskDrive -VMName <VM name>
+  ```
+
+4. Make sure the VM disk has the correct format.   
+  - Use a **dynamic disk** format, as the **static format is not compatible with XCP-ng**.
+  - If the disk is in the **VHDX** format, convert it to the **VHD** format. 
+  To convert the disk from VHDX to VHD, run this command:  
+
+  ```powershell
+  Convert-VHD -Path <source path> -DestinationPath <destination path> -VHDType Dynamic
+  ```
+  - Name the VHD file using the `<UUID>.vhd` pattern. The file extension is critical for correctly identifying the disk type.
```

Comment:
It's too soon to say that there must be a UUID in the file name. The extension is important, but at this stage the file name isn't.

In the next sections, the user will have to choose between importing through Xen Orchestra's UI, or by doing a direct copy to XCP-ng.